src/arr/compiler/type-check-structs.arr

```diff
@@ -829,8 +829,10 @@ fun solve-helper-examples(system :: ConstraintSystem, solution :: ConstraintSolu
   cases(ConstraintSystem) system:
     | no-constraints => fold-result({system; solution}, context)
     | constraint-system(variables, constraints, refinement-constraints, field-constraints, example-types, next-system) =>
-      foldr-fold-result(lam(existential-key, shadow context, {shadow system; shadow solution}):
+      foldr-fold-result(lam(existential-key, shadow context, {shadow system; shadow solution}) block:
         {existential; _; fun-examples; checking-fun} = example-types.get-value(existential-key)
+        print("existential: " + tostring(existential) + "\n")
+        print("fun-examples: " + tostring(fun-examples) + "\n")
```

Comment:
On CPO, won't these `print` calls result in those messages being printed to the REPL?